source/include/Variants.h

```diff
@@ -59,6 +59,9 @@ class VariantAttributeManager {
         CleanUp();
     }
 };
+
+enum { kVariantArgErr = 1, kVariantIncompatibleType = 91};
```

Comment:
I think this is fine for now. But, please create a github issue to consolidate all LV errors to a header file.

Reply:
Good point. There is already an issue for this: https://github.com/ni/VireoSDK/issues/384

Reply:
There was already an issue for this: https://github.com/ni/VireoSDK/issues/384